src/openforms/submissions/api/serializers.py

```diff
@@ -155,6 +155,14 @@ class SubmissionSerializer(serializers.HyperlinkedModelSerializer):
         write_only=True,
         default=False,  # type: ignore
     )
+    object_reference = serializers.CharField(
+        label=_("Object reference"),
+        required=False,
+        help_text=_(
+            "The object reference-id which will be used to modify the product in the registration backend."
```

Comment:
```suggestion
            "The object reference-id which will be used to modify the product in the registration backend. "
```

Reply:
But I think adding `object_reference` to `Meta.fields` is enough?

Reply:
Yes you are right, I am fixing that now.